### PR TITLE
MRG: Move compare_matrix to apps/tools

### PR DESCRIPTION
**Pull request by massich**
requires #374 to have smaller diff.

### CODE REVIEW

**Comment by agramfort** [comment]
you need to rebase.

can you try to remove dep on OpenMEEG::OpenMEEG ?

**Comment by massich** [comment]
`compare_matrix.cpp` uses `"options.h" to handle the arguments. This belongs to `OpenMEEG::OpenMEEG`

File: apps/tools/CMakeLists.txt [hunk 1]
```diff
@@ -37,6 +37,10 @@ target_link_libraries(om_check_geom OpenMEEG ${VTK_LIBRARIES})
 add_executable(om_mesh_to_dip mesh_to_dip.cpp)
 target_link_libraries(om_mesh_to_dip OpenMEEG ${VTK_LIBRARIES})
 
+add_executable(compare_matrix compare_matrix.cpp)
```
Comment (agramfort) [inline apps/tools/CMakeLists.txt]: maybe we call it test_compare_matrix to be consistent? I guess it's not installed/shipped anyway so we can rename 

I would do the same for test_load_geo

Comment (massich) [reply apps/tools/CMakeLists.txt]: yes we have several of them. We can change them all at once in a separated PR.

I guess that `validateEIT` also falls in this category

**Comment by agramfort** [comment]
green. Merging

**Pull request merged**